### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## Problem
- *Global upgrade of the package.*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Kay0u
- [x] **Approval (LGTM)** : Kay0u
- [x] **Approval (LGTM)** : JimboJoe
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/strut_ynh%20testing%20(Official)/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by maniackcrudelis** [comment]
Should we wait as well for real changes before notifying this testing ?
Is it going to be any...?

**Comment by kay0u** [comment]
> Should we wait as well for real changes before notifying this testing ?
> Is it going to be any...?

The question is: Is this app going to receive any real change?

**Comment by maniackcrudelis** [comment]
Don't think we should expect any more change soon.
Ready for reviews

File: scripts/upgrade [hunk 1]
```diff
@@ -6,113 +6,148 @@
...
 # Create a dedicated nginx config
 ynh_add_nginx_config
 
+#=================================================
+# SPECIFIC UPGRADE
+#=================================================
+# PATCH SOURCE
+#=================================================
+
+if [ "$upgrade_type" == "UPGRADE_APP" ]
```
Comment (alexAubin) [inline scripts/upgrade]: Isn't this the same condition as before which already contains the same patch about the imgur thing ?

Comment (maniackcrudelis) [reply scripts/upgrade]: Indeed, the code was duplicated

Comment (kay0u) [reply scripts/upgrade]: Good catch!

**Comment by kay0u** [comment]
Bump

**Comment by JimboJoe** [comment]
Can be merged in 3 days.

**Pull request merged**